server/base/src/main/java/org/apache/accumulo/server/master/balancer/ChaoticLoadBalancer.java

```diff
@@ -52,17 +52,10 @@
 public class ChaoticLoadBalancer extends TabletBalancer {
   private static final Logger log = LoggerFactory.getLogger(ChaoticLoadBalancer.class);
 
-  @SuppressWarnings("unused")
-  private final String tableName;
-
-  public ChaoticLoadBalancer() {
-    this.tableName = null;
-  }
+  public ChaoticLoadBalancer() {}
 
   // Required constructor
-  public ChaoticLoadBalancer(String tableName) {
-    this.tableName = tableName;
-  }
+  public ChaoticLoadBalancer(String tableName) {}
```

Comment:
Out of curiosity, why is this constructor required?

Reply:
We dynamically load balancer classes, based on user configuration. My guess is that this supports per-table balancing features. I left it as-is, rather than head down the rabbit hole of figuring out if it was necessary. :smiley_cat: